consensus/polybft/signer/common.go

```diff
@@ -0,0 +1,139 @@
+package bls
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"math/big"
+
+	bn256 "github.com/umbracle/go-eth-bn256"
+)
+
+var (
+	zeroBigInt = *big.NewInt(0)
+	oneBigInt  = *big.NewInt(1)
+	g2         = *new(bn256.G2).ScalarBaseMult(&oneBigInt)  // Generator point of G2 group
+	zeroG1     = *new(bn256.G1).ScalarBaseMult(&zeroBigInt) // Zero point in G2 group
+	zeroG2     = *new(bn256.G2).ScalarBaseMult(&zeroBigInt) // Zero point in G2 group
+
+	// this is a copy of point p is a prime over which we form a basic field: 36u⁴+36u³+24u²+6u+1.
+	// TODO delete this point and use bn256.p
+	bn256P, _ = new(big.Int).SetString("21888242871839275222246405745257275088696311157297823662689037894645226208583", 10)
+)
+
+// genRandomBytes generates byte array with random data
+func genRandomBytes(size int) (blk []byte) {
+	blk = make([]byte, size)
+
+	_, err := rand.Reader.Read(blk)
+	if err != nil {
+		panic("Cannot generate message for tests. This should never happen")
+	}
+
+	return
+}
+
+// leftPadTo32Bytes appends zeros to bytes slice to make it exactly 32 bytes long
+func leftPadTo32Bytes(bytes []byte) ([]byte, error) {
```

Comment:
You don't need to define a method for this, you can use `PadLeftOrTrim` from the `common` package